packages/twenty-server/.env.example

```diff
@@ -74,4 +74,8 @@ FRONTEND_URL=http://localhost:3001
 # SSL_CERT_PATH="./certs/your-cert.crt"
 # CLOUDFLARE_API_KEY=
 # CLOUDFLARE_ZONE_ID=
-# CLOUDFLARE_WEBHOOK_SECRET=
\ No newline at end of file
+# CLOUDFLARE_WEBHOOK_SECRET=
+# CLICKHOUSE_URL=
```

Comment:
Is it possible to have a single connection string to be close to what we do for Redis and Postgres? https://github.com/ClickHouse/ClickHouse/pull/50689

Reply:
Fix in https://github.com/twentyhq/twenty/pull/11174/commits/2566a7cb1a142e169835c416134878be34973bcc